packages/nx-plugin/src/executors/autorun/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { globalConfig, persistConfig, uploadConfig } from '../internal/config';
+import { NormalizedExecutorContext } from '../internal/context';
+import {
+  AutorunCommandExecutorOnlyOptions,
+  AutorunCommandExecutorOptions,
+} from './schema';
+
+export function parseAutorunExecutorOnlyOptions(
+  options: Partial<AutorunCommandExecutorOnlyOptions>,
+): AutorunCommandExecutorOnlyOptions {
+  const { projectPrefix, dryRun, onlyPlugins } = options;
+  return {
+    ...(projectPrefix == null ? {} : { projectPrefix }),
+    ...(dryRun == null ? {} : { dryRun: dryRun }),
+    ...(onlyPlugins ? { onlyPlugins } : {}),
```

Comment:
```suggestion
    ...(projectPrefix && { projectPrefix }),
    ...(dryRun != null && { dryRun: dryRun }),
    ...(onlyPlugins && { onlyPlugins }),
```

Reply:
Nice change! reads better and more compact